### DISCUSSION TITLE
Does the Linux version work for ARM chipsets (raspberry pi 5)?

### DISCUSSION THREAD

**Discussion by cmorenoserrano** [K3vinb5/Unyo#50] [Q&A]
I'm trying to get Unyo to work on my raspberry pi 5, but the AppImage does not seem to work.
I wonder if the image has been built for Intelx86 or ARM64bit chipsets.
I just want confirmation as I'm losing my mind trying to get it to work.
Many thanks in advance.

**Comment by K3vinb5** [comment]
Hi! So, I haven't compiled it to arm chips on linux yet, But I could do it. Right now I have literally no time as you and probably a bunch of people have seen by the commits but once the new year starts and life slows down a bit I'll make sure to get new features releasing and as you suggested compile it to arm on linux (and windows) as well. I would just ask you to please open a ticket with that issue so that I don't end up forgeting.

**Reply by cmorenoserrano** [reply]
Thanks! I'll open an issue then. :) I'm currently using the Mac one and it works beautifully.

**Comment by K3vinb5** [comment]
I will make that a priority since I will need to do that on windows anyways, and the pipeline is needing some work anyway.
